cmd/package_managers.go

```diff
@@ -18,12 +18,46 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"regexp"
 
 	ngt "github.com/ossf/scorecard/v4/cmd/internal/nuget"
 	pmc "github.com/ossf/scorecard/v4/cmd/internal/packagemanager"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+var _GITHUB_DOMAIN_REGEXP = regexp.MustCompile(`^https?://github[.]com/([^/]+)/([^/.]+)`)
+var _GITHUB_SUBDOMAIN_REGEXP = regexp.MustCompile(`^https?://([^.]+)[.]github[.]io/([^/.]+).*`)
```

Comment:
`.` is a valid character in a github repo name. (e.g. https://github.com/0k/kids.cmd). I assume this is to strip the `.git` suffix? Perhaps do that at the end of `makeGithubRepo`?

Reply:
Good call, done